test/integration-test/commands/generate-application/generate-application.test.ts

```diff
@@ -0,0 +1,54 @@
+import { expect, test } from '@oclif/test'
+import fs from 'fs'
+
+const ticketing = 'ticketing'
+const education = 'education'
+const health = 'health'
+const portableReputation = 'portable-reputation'
+
+describe('User generates application', () => {
+  after(() => {
+    fs.rmSync(`./${ticketing}`, { recursive: true, force: true })
+    fs.rmSync(`./${education}`, { recursive: true, force: true })
+    fs.rmSync(`./${health}`, { recursive: true, force: true })
+    fs.rmSync(`./${portableReputation}`, { recursive: true, force: true })
+  })
+  describe('Generate ticketing app', () => {
+    test
+      .stdout()
+      .command(['generate-application', '-n', `${ticketing}`, '-u', `${ticketing}`])
+      .it('generate ticketing app', () => {
+        expect(fs.existsSync(`./${ticketing}`)).be.true
```

Comment:
could you please check also package.json -> name field for all 4 cases ?

would be great to be sure that we have cloned the correct repo